mozartdata/transforms/dim/netsuite_shipping_address.sql

```diff
@@ -0,0 +1,157 @@
+with base as (
+  select
+    a.nkey
+  , 'invoice' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.invoiceshippingaddress a
+  where
+    _fivetran_deleted = false
+  union all
+  select
+    a.nkey
+  , 'cashsale' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.cashsaleshippingaddress a
+  where
+    _fivetran_deleted = false
+  union all
+  select
+    a.nkey
+  , 'cashrefund' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.cashrefundshippingaddress a
+  where
+    _fivetran_deleted = false
+  union all
+  select
+    a.nkey
+  , 'salesorder' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.salesordershippingaddress a
+  where
+    _fivetran_deleted = false
+  union all
+  select
+    a.nkey
+  , 'itemfulfillment' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.itemfulfillmentshippingaddress a
+  where
+    _fivetran_deleted = false
+  union all
+  select
+    a.nkey
+  , 'purchaseorder' as record_type
+  , a.addr1
+  , a.addr2
+  , a.addr3
+  , a.addressee
+  , a.addrphone
+  , a.addrtext
+  , a.attention
+  , a.city
+  , a.state
+  , a.country
+  , a.zip
+  , a.override
+  , a.dropdownstate
+  from
+    netsuite.purchaseordershippingaddress a
+  where
+    _fivetran_deleted = false
+)
+
+select
+    concat(b.record_type,'_',b.nkey) as shipping_address_id_edw
+  , b.nkey as shipping_address_id_ns
+  , b.addr1 as address_1
+  , b.addr2 as address_2
+  , b.addr3 as address_3
+  , b.addressee as customer_name
+  , b.addrphone as phone_number
+  , NULLIF(REGEXP_REPLACE(REGEXP_REPLACE(b.addrphone, '^1|\\+1\\s?|\\(|\\)|-|\\s', ''),'[^0-9]',''), '') as normalized_phone_number
+  , b.addrtext as address_text
+  , b.attention
+  , b.city
+  , coalesce(sa.fullname, b.state) as state
+  , coalesce(sf.shortname, b.state) as state_abbreviation
```

Comment:
Similar to above, shouldn't this use the join to the shortname table?
```suggestion
  , coalesce(sa.shortname, b.state) as state_abbreviation
```